bindings/ergo-lib-wasm/src/ergo_box.rs

```diff
@@ -163,10 +198,16 @@ impl BoxValue {
         BoxValue(chain::ergo_box::BoxValue::SAFE_USER_MIN)
     }
 
-    /// Create from u32 with bounds check
-    pub fn from_u32(v: u32) -> Result<BoxValue, JsValue> {
+    /// Number of units inside one ERGO (i.e. one ERG using nano ERG representation)
+    #[allow(non_snake_case)]
+    pub fn UNITS_PER_ERGO() -> I64 {
+        (chain::ergo_box::BoxValue::UNITS_PER_ERGO as i64).into()
+    }
+
+    /// Create from i64 with bounds check
+    pub fn from_i64(v: &I64) -> Result<BoxValue, JsValue> {
```

Comment:
I got rid of the `from_u32` here as per our discussion that it was better to replace it with `from_i64`